packages/calcite-design-tokens/support/types/tokenTypes/colorScheme.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint @cspell/spellchecker: 0 */
```

Comment:
Can we make sure to disable at the enum level for unrecognized entries and not the whole file?

Applies to other disabled spell checker lines.